src/main/io/ledstrip.c

```diff
@@ -543,10 +549,33 @@ static void applyLedFixedLayers(void)
             break;
 
         case LED_FUNCTION_BATTERY:
+        case LED_FUNCTION_BATTERY_BAR:
```

Comment:
```suggestion
        case LED_FUNCTION_BATTERY_BAR: {
            if (ledBarIdx[LED_BAR_BATTERY] < (calculateBatteryPercentageRemaining() * ledBarCount[LED_BAR_BATTERY] + 50) / 100) {
                   color = HSV(RED);
                   hOffset += MAX(scaleRange(calculateBatteryPercentageRemaining(), 0, 100, -30, 120), 0);
            } else {
                  color = getSC(LED_SCOLOR_BACKGROUND));
            }
            ledBarIdx[LED_BAR_BATTERY]++;
            break;
        case LED_FUNCTION_BATTERY:
```

But no led would be on for small `calculateBatteryPercentageRemaining()` (after rounding). It would be better to interpolate color

Reply:
Now, at least one LED is lit up at all times. This shows the interpolating color even when using just 1 led.